hcn/hcnroute.go

```diff
@@ -0,0 +1,252 @@
+package hcn
+
+import (
+	"encoding/json"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+	"github.com/Microsoft/hcsshim/internal/interop"
+	"github.com/sirupsen/logrus"
+)
+
+// HostComputeRoute represents SDN routes.
+type HostComputeRoute struct {
+	ID                   string                  `json:"ID,omitempty"`
+	HostComputeEndpoints []string                `json:",omitempty"`
```

Comment:
Should this be a string? I assume these are guids?

Reply:
The schema (i.e. HNS.Schema.mars) says its a []string, even though they might actually be guids. Also the other implementation (e.g. load balance) has a similar pattern. Let me know what you think.